### PR TITLE
platform: add OVHcloud

### PR DESCRIPTION
**Pull request by tormath1**
Everything is in the title.

Related to: https://github.com/flatcar/Flatcar/issues/1453

### CODE REVIEW

**Review by krnowak — DISMISSED** [review]

**Review by till — APPROVED** [review]
Looks gut to me 😁

Kinda wild that OpenStack is the only true multi cloud environment where you can use the same tools across providers. 🫣

File: content/docs/latest/_index.md [hunk 1]
```diff
@@ -30,6 +30,7 @@ platforms and bare metal servers.
  * [Hetzner][hetzner]
  * [OpenStack][openstack]
  * [Brightbox][brightbox]
+ * [OVH Cloud][ovhcloud] (community support)
```
Comment (sbraz) [inline content/docs/latest/_index.md:33]: Just a nit but
```suggestion
 * [OVHcloud][ovhcloud] (community support)
```
(I'm only mentioning it once but there are other instances).

File: content/docs/latest/installing/community-platforms/ovhcloud.md [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+
+These instructions will walk you through using Flatcar on [OVH Cloud][ovhcloud], importing an image, and running your first server using the OpenStack command line interface. Please note that OVH Cloud is a community supported platform at the moment which means that the CI does not run on OpenStack images in OVH Cloud environment.
+
+Additional note: OVH Cloud is compatible with OpenStack: it is not a mistake if we refer to OpenStack images and specific variables.
+
+## Import the image
+
+_Note_: This getting started assumes that OpenStack CLI is already installed and properly configured (see OVH Cloud [documentation][ovhcloud-documentation-import])
+
+It is possible to import your own Flatcar image in OVH Cloud using the OpenStack CLI. Here's example, to upload an image from the Flatcar Beta channel:
```
Comment (sbraz) [inline content/docs/latest/installing/community-platforms/ovhcloud.md:15]:
```suggestion
It is possible to import your own Flatcar image in OVH Cloud using the OpenStack CLI. Here's an example, to upload an image from the Flatcar Beta channel:
```

File: content/docs/latest/installing/community-platforms/ovhcloud.md [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+```bash
+$ wget https://beta.release.flatcar-linux.net/amd64-usr/current/flatcar_production_openstack_image.img
+$ openstack image create \
+  --container-format bare \
+  --disk-format qcow2 \
+  --property image_original_user=core \
+  --property hw_qemu_guest_agent=yes \
+  --file flatcar_production_openstack_image.img \
+  flatcar-beta
+$ IMAGE_ID=$(openstack image show flatcar-beta --format json | jq -r .id)
```
Comment (sbraz) [inline content/docs/latest/installing/community-platforms/ovhcloud.md:26]: I don't have OpenStack on my PC here but can't we do this?
```suggestion
$ IMAGE_ID=$(openstack image show flatcar-beta --format value --column id)
```

File: content/docs/latest/installing/community-platforms/ovhcloud.md [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+```
+
+## Using Flatcar Container Linux
+
+Now that you have a machine booted it is time to play around. Check out the [Flatcar Container Linux Quickstart][quickstart] guide or dig into [more specific topics][doc-index].
+
+[butane-configs]: ../../provisioning/config-transpiler
+[doc-index]: ../../
+[ovhcloud]: https://www.ovhcloud.com/en/
+[ovhcloud-documentation-create]: https://support.us.ovhcloud.com/hc/en-us/articles/360002245164-Creating-and-Connecting-a-Public-Cloud-Instance
```
Comment (sbraz) [inline content/docs/latest/installing/community-platforms/ovhcloud.md:142]:
```suggestion
[ovhcloud-documentation-create]: https://help.ovhcloud.com/csm/en-public-cloud-compute-getting-started?id=kb_article_view&sysparm_article=KB0051009
```
As the US documentation is handled separately from the rest, it's best to refer to the worldwide one.

File: content/docs/latest/installing/community-platforms/ovhcloud.md [hunk 5]
```diff
@@ -0,0 +1,144 @@
...
+        Restart=always
+        RestartSec=5s
+        [Install]
+        WantedBy=multi-user.target
+```
+
+Transpile it to Ignition JSON:
+
+```bash
+cat butane.yaml | docker run --rm -i quay.io/coreos/butane:release > ignition.json
```
Comment (sbraz) [inline content/docs/latest/installing/community-platforms/ovhcloud.md:73]: We could avoid cat here
```suggestion
docker run --rm -i quay.io/coreos/butane:release < butane.yaml > ignition.json
```

File: content/docs/latest/installing/community-platforms/ovhcloud.md [hunk 6]
```diff
@@ -0,0 +1,144 @@
...
+  --image "${IMAGE_ID}" \
+  --format json \
+  flatcar-01 | jq -r .id)
+```
+
+Your first Flatcar instance should now be running. At this point, the instance should be visible in the OVH Cloud portal too. The only thing left to do is find the IP address and SSH in.
+
+Example:
+```shell
+$ openstack server show "${INSTANCE_ID}" --format json | jq -r .addresses
```
Comment (sbraz) [inline content/docs/latest/installing/community-platforms/ovhcloud.md:110]: Can we use `--format value` as well here? I wonder.

**Review by sbraz — COMMENTED** [review]
Thanks a lot for this :) I'll ask some of my colleagues who work on the public cloud team to look into it as well.

File: content/docs/latest/installing/community-platforms/ovhcloud.md (hunk 3 above)
Comment (sbraz) [reply content/docs/latest/installing/community-platforms/ovhcloud.md:26]: Actually you could just pass these to `openstack image create` and skip the `image show` command which could behave weirdly (fail?) if two images are named `flatcar-beta`.

**Review by krnowak — APPROVED** [review]

**Pull request merged**